Libplanet/Blockchain/BlockChain.cs

```diff
@@ -564,9 +564,11 @@ bool renderActions
 
                 string ns = Id.ToString();
                 HashDigest<SHA256>? tip = Store.IndexBlockHash(ns, -1);
+                IEnumerable<Transaction<T>> transactions =
+                    block.Transactions.OrderBy(tx => tx.Nonce);
```

Comment:
Is this order intended?

Reply:
Yes, it's to pass validation of tx nonce.